### PR TITLE
IT recruitment update

### PR DESCRIPTION
**Pull request by kvas-damian**

### CODE REVIEW

File: src/content/jobs/full-stack-ai-lead.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+meta:
+    title: Fullstack AI Lead - Dołącz do Bethink!
+    description: Świetny developer dla każdego - starego i młodego
+    linkImageUrl: asdfasdf
```
Comment (rogatty) [inline src/content/jobs/full-stack-ai-lead.md:5]: What about this? Same for heroImage

File: src/content/jobs/full-stack-ai-lead.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+---
+
+Nowoczesny stack, doświadczony zespół i praca nad autorską platformą e-learningową.
+
+Rozwijaj aplikację, na której prowadzone są największe w Polsce medyczne i maturalne kursy online.
+
+### Gdzie, z kim, za ile?
+
+-   📍 Praca remote lub w biurze (Poznań, Podlaska 15, Sołacz)
+-   💵 29.3 - 41.8k brutto UOP (80% koszty uzyskania przychodu) / 29.3 - 41.8k netto B2B
```
Comment (rogatty) [inline src/content/jobs/full-stack-ai-lead.md]: It's inconsistent with salaryRange in frontmatter.

Comment (kvas-damian) [reply src/content/jobs/full-stack-ai-lead.md]: removed

File: src/content/jobs/full-stack-ai-lead.md (hunk 1 above)
Comment (kvas-damian) [reply src/content/jobs/full-stack-ai-lead.md:5]: Copied from existing job offer. It isn't used anywhere now. Don't want to clean it now

**Review by rogatty — APPROVED** [review]

**Comment by adamkarminski** [comment]
@rogatty @kvas-damian - respect 😄

**Pull request merged**